pages/common/virsh-connect.md

```diff
@@ -0,0 +1,21 @@
+# virsh-connect
+
+> Connect to a virtual machine hypervisor.
+> See also: `virsh`.
+> More information: <https://manned.org/virsh>.
+
+- Connect to the default hypervisor:
+
+`virsh connect`
+
+- Connect as root to the local QEMU/KVM hypervisor:
+
+`virsh connect qemu:///system`
+
+- Launch a new instance of the hypervisor and connect to it as a local user:
```

Comment:
'a' or 'the'? 🤔 
```suggestion
- Launch a new instance of the hypervisor and connect to it as the local user:
```

Reply:
The libvirt FAQ uses "your" : https://wiki.libvirt.org/page/FAQ#What_is_the_difference_between_qemu:.2F.2F.2Fsystem_and_qemu:.2F.2F.2Fsession.3F_Which_one_should_I_use.3F

Reply:
Hmm, but we're [not supposed to use personal pronouns](https://github.com/tldr-pages/tldr/blob/8939f2e0eb85647a75a20026281bd503614fa855/CONTRIBUTING.md#inclusive-language) like that...

Reply:
The only reference to inclusive language that I could find in the guidelines is to not use gender specific words. It suggests using 'they' instead of 'him/her.

Reply:
You know, you're right about that.  But @sbrl has said before that we shouldn't use 2nd person, e.g. [here](https://github.com/tldr-pages/tldr/pull/5819#discussion_r619688683).